packages/runtime/test/consumption/attributes.test.ts

```diff
@@ -611,6 +655,35 @@ describe(CreateRepositoryAttributeUseCase.name, () => {
         await expect(services1.eventBus).toHavePublished(AttributeCreatedEvent, (e) => e.data.content.value["@type"] === "City");
         await expect(services1.eventBus).toHavePublished(AttributeCreatedEvent, (e) => e.data.content.value["@type"] === "Country");
     });
+
+    test("should create a RepositoryAttribute that is the default if it is the first of its value type", async () => {
+        const request: CreateRepositoryAttributeRequest = {
+            content: {
+                value: {
+                    "@type": "Pseudonym",
+                    value: "Pseudo Petra"
```

Comment:
This makes me laugh a lot :D But please use a more descriptive value here, for example "A pseudonym".